packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/Label.tsx

```diff
@@ -53,7 +56,7 @@ export const Label = memo(({ diagramElementId, label, faded, transform }: LabelP
   };
 
   const content: JSX.Element =
-    label.id === currentlyEditedLabelId ? (
+    label.id === currentlyEditedLabelId && !readOnly ? (
```

Comment:
Can we even have a `currentlyEditedLabelId` in `readOnly`?

Reply:
It seems it cannot